src/MockManager.php

```diff
@@ -343,6 +346,8 @@ function(
 	}
 
 	protected static function unmock(string $mockKey): void {
+    /* HH_FIXME[2049] This function is not in any hhi. */
+    /* HH_FIXME[4107] This function is not in any hhi. */
```

Comment:
We can follow up on these indentation issues in a PR that introduces `hackfmt`.

Reply:
Okay, I don't feel I deserve credit if hackfmt formats about 1/3 of the repo. In case we don't mind that I end up claiming all the fame/(git blame), I can run my formatter. If the needs to be done by a bot, you can feed it this script:

```SH
#!/usr/bin/env bash

for f in $(find {src,tests} -name '*.php' -or -name '*.hack'); do 
  if ! grep -q "SignedSource<<" "$f"; then
    hackfmt $f -i
  fi
done
```
from /path/to/quizlet/hackfmt.

It

Reply:
We don't really care about who gets the credit / `git blame`s here, just wanna make something useful. Feel free to open a large formatting PR. 🙂